cmd/re/main.go

```diff
@@ -0,0 +1,259 @@
+//
```

Comment:
Kill these blank `//`, do similar as in other services: https://github.com/mainflux/mainflux/blob/master/http/adapter.go